### PR TITLE
:sparkles: Improve state badges in front-end

### PR DESCRIPTION
**Pull request by acco**

### CODE REVIEW

**Comment by acco** [comment]
* **#698** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/698?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/698?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: lib/sequin_web/live/sink_consumers/show.ex [hunk 1]
```diff
@@ -824,22 +829,38 @@ defmodule SequinWeb.SinkConsumersLive.Show do
...
+      _ -> "gray-200"
+    end
+  end
+
+  defp get_message_state_color(_consumer, state) do
+    case state do
+      "delivered" -> "green-200"
+      "backing off" -> "yellow-200"
+      _ -> "gray-200"
     end
```
Comment (RTLS) [inline lib/sequin_web/live/sink_consumers/show.ex:864]: is this safe? tailwind scans css, js, .svelte files etc. for tailwind classes that are used and only builds for those classes

does it scan .ex files? I'm sure we can have it do so

File: assets/svelte/consumers/ShowMessages.svelte [hunk 2]
```diff
@@ -597,7 +577,7 @@
                       </div>
                       <div class="flex justify-between items-center">
                         <span class="text-sm font-medium text-gray-500"
-                          >Not Visible Until:</span
+                          >Backoff Until:</span
```
Comment (RTLS) [inline assets/svelte/consumers/ShowMessages.svelte:580]: nice

**Review by RTLS — APPROVED** [review]
missing the migration?

**Pull request merged**